src/Docker/Runner.php

```diff
@@ -305,8 +305,9 @@ private function runRow(
                 $jobScopedEncryptor
             );
         } catch (\Exception $e) {
-            $dataLoader->cleanWorkspace();
             throw $e;
+        } finally {
+            $dataLoader->cleanWorkspace();
```

Comment:
No tímhle si vůbec nejsem jistej, to je příliš brzo, prože normální cleanup je tu https://github.com/keboola/docker-bundle/blob/9087fe8c8ea49881732d518fd7294bcf5c2495f7/src/Docker/Runner.php#L425
nad tím se ještě budu muset zamyslet

Reply:
jo, taky nejsem byl o tom jiste